pyspecProcScripts/first_level/fl_dummy.py

```diff
@@ -1,14 +0,0 @@
-"""provides object fl_dummy that can take the place of a figure list, but doesn't do anything.
```

Comment:
I had said don't delete this but it's good to delete -- this was moved to pyspecdata